### PR TITLE
chore(deps): Bump core version

### PR DESCRIPTION
**Pull request by tlabaj**
closes: #8831 

Bump core version

### CODE REVIEW

File: packages/react-core/src/components/DatePicker/DatePicker.tsx [hunk 1]
```diff
@@ -272,7 +272,7 @@ const DatePickerBase = (
         </div>
       </Popover>
       {helperText && <div className={styles.datePickerHelperText}>{helperText}</div>}
-      {errorText.trim() && <div className={css(styles.datePickerHelperText, styles.modifiers.error)}>{errorText}</div>}
+      {errorText.trim() && <div className={css(styles.datePickerHelperText)}>{errorText}</div>}
```
Comment (gitdallas) [inline packages/react-core/src/components/DatePicker/DatePicker.tsx:275]: Won't this cause error to be unstyled (or styled the same way as helper)?

Comment (gitdallas) [reply packages/react-core/src/components/DatePicker/DatePicker.tsx:275]: image

looks like it is also causing cypress to fail as it tries to find `div.pf-m-error` 
https://github.com/patternfly/patternfly-react/blob/v5/packages/react-integration/cypress/integration/datepicker.spec.ts#L17

Comment (tlabaj) [reply packages/react-core/src/components/DatePicker/DatePicker.tsx:275]: There is a follow up issue for the helper text work

Comment (tlabaj) [reply packages/react-core/src/components/DatePicker/DatePicker.tsx:275]: I disabled the test for now.  It should be reenabled with the datePicker helper text work (I left a note in the issue).  This PR is just to bump the core version and get it to build so we have the styles to address the feature work once this goes in

**Review by gitdallas — APPROVED** [review]

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**